lib/configs/workiva.dart

```diff
@@ -0,0 +1,17 @@
+/// An opinionated `tool/dev.dart` base configuration intended to help
+/// standardize dart_dev configuration and command-line usage across Dart
+/// projects in the Workiva organization.
+library dart_dev.configs.workiva;
```

Comment:
Wasn't this moving to its own package or something?

Reply:
Yes, I haven't pushed up the commit to the other repo, yet. But when I do, it will be pretty much a direct move from this repo to dart_dev_workiva

Reply:
Moved.